otlp/common.go

```diff
@@ -462,10 +462,13 @@ func addAttributeToMap(ctx context.Context, result map[string]interface{}, key s
 // helps reduce allocation and copying.
 // Note that an Encoder emits JSON with a trailing newline because it's intended for use
 // in streaming. This is correct but sometimes surprising and the tests need to expect it.
+// NOTE: The JSON encoder has HTML escaping disabled.
```

Comment:
not a huge deal but this is the second note and the formatting doesn't match the previous `Note` comment